docs/Developer-Docs/GoldenGate-and-Driver-Development.rst

```diff
@@ -154,3 +154,10 @@ Defining A New Test
 Extend the appropriate ScalaTest spec or base class, and
 place the file under the correct ``src/test/scala`` directory. They will be
 automatically enumerated by ScalaTest and will run in CI by default.
+
+C/C++ guidelines
+++++++++++++++++
+
+The C++ sources follow the LLVM style guide and should be formatted accordingly
+using `clang-format`. `git clang-format` can be used before committing to ensure
+that files are properly formatted. Formatting is enforced through a CI job.
```

Comment:
```suggestion
The C++ sources are formatted using ``clang-format`` and all submitted pull-requests 
must be formatted following `https://github.com/firesim/firesim/blob/main/.clang_format`_ 
prior to being accepted and merged.

``git clang-format`` can be used before committing to ensure that files are properly formatted. 
```

RST inline literals need to be surrounded in double backtics.

Should explain that we're using `clang-format` in a way that doesn't imply that we've adopted something called the "LLVM style guide" that lives somewhere like https://llvm.org/docs/CodingStandards.html or something else that a search engine might confuse someone with.